content/03.ml-concepts.md

```diff
@@ -2,15 +2,23 @@
 Deep learning is a distinct subfield of machine learning, but it is still a subfield.
 DL has proven to be an extremely powerful paradigm capable of outperforming “traditional” machine learning approaches in certain contexts, but it is not immune to the many limitations inherent to machine learning.
 Many best practices for machine learning also apply to deep learning.
-For instance, deep supervised learning models should be trained, tuned, and tested on non-overlapping datasets.
```

Comment:
This section seemed to be a list but I couldn't really nail down themes and it felt like it jumped back and forth across areas to me. So I tried to separate data and metrics. I also wasn't really sure what to say about the extreme cases & there were no examples cited. I think that would really benefit from either some more work, or else consider removing.